.github/workflows/release.yaml

```
@@ -0,0 +1,64 @@
+name: Release
+
+on:
+    workflow_dispatch:
+        inputs:
+            release-version:
+                type: string
+                required: true
+                description: 'Version number (for example: v0.0.4)'
+            codeflare-operator-version:
+                type: string
+                required: true
+                description: "Version of Codeflare Operator (for example v0.0.4)"
+            multi-cluster-app-dispatcher-version:
+                type: string
+                required: true
+                description: "Version of Multi-Cluster App Dispatcher (for example v1.31.0)"
+            codeflare-sdk-version:
+                type: string
+                required: true
+                description: "Version of CodeFlare-SDK (for example v0.4.4)"
+            instascale-version:
+                type: string
+                required: true
+                description: "Version of InstaScale (for example v0.0.4)"
+            kuberay-version:
+                type: string
+                required: true
+                description: "Version of KubeRay (for example v0.5.0)"
+
+jobs:
+    release:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: write
+        steps:
+            - name: Checkout the repository
+              uses: actions/checkout@v3
+            - name: Update codeflare-operator version in tests/resources/codeflare-subscription.yaml file
+              uses: fjogeleit/yaml-update-action@main
+              with:
+                valueFile: "tests/resources/codeflare-subscription.yaml"
+                propertyPath: "spec.startingCSV"
+                value: "codeflare-operator.${{ github.event.inputs.codeflare-operator-version }}"
```

A second thought is using `yq` command:
```suggestion
            - name: Update codeflare-operator version in tests/resources/codeflare-subscription.yaml file
              run: |
                yq -i '.spec.startingCSV = "codeflare-operator.${{github.event.inputs.codeflare-operator-version}}"' tests/resources/codeflare-subscription.yaml
```